drivers/hmis/app/graphql/types/hmis_schema/custom_data_element.rb

```diff
@@ -8,27 +8,31 @@
 
 module Types
   class HmisSchema::CustomDataElement < Types::BaseObject
-    # object is an OpenStruct, see HasCustomDataElements
-
     field :id, ID, null: false
     field :key, String, null: false
+    field :field_type, HmisSchema::Enums::CustomDataElementType, null: false
     field :label, String, null: false
     field :repeats, Boolean, null: false
-    field :value, HmisSchema::CustomDataElementValue, null: true
-    field :values, [HmisSchema::CustomDataElementValue], null: true
+    field :at_occurrence, Boolean, null: false
+    field :value, HmisSchema::CustomDataElementValue, null: true, extras: [:parent]
+    field :values, [HmisSchema::CustomDataElementValue], null: true, extras: [:parent]
 
-    # If this custom element only allows one value, 'value' is set
-    def value
-      return if object.repeats
+    # object is a CustomDataElementDefinition
+    # parent is the parent (Enrollment, Client, etc)
 
-      object.values&.first
+    def all_values(parent:)
+      parent = parent.owner if parent.is_a? Hmis::Hud::HmisService # special case for view
```

Comment:
assuming parent.owner is an AR relation that's not eager loaded, we'd probably want this
```suggestion
      parent = load_ar_association(parent, :owner) if parent.is_a? Hmis::Hud::HmisService # special case for view
```